code/game/gamemodes/modes_gameplays/changeling/powers/Chameleon_skin.dm

```diff
@@ -51,10 +51,18 @@
 	var/datum/role/changeling/C = owner.mind.GetRoleByType(/datum/role/changeling)
 	C.chem_recharge_slowdown -= 0.25
 	owner.invisibility = INVISIBILITY_NONE
+	var/datum/atom_hud/M = global.huds[DATA_HUD_MEDICAL]
+	M.add_to_hud(owner)
+	var/datum/atom_hud/S = global.huds[DATA_HUD_SECURITY]
+	S.add_to_hud(owner)
 
 /obj/effect/proc_holder/changeling/chameleon_skin/proc/turn_on()
 	to_chat(owner, "<span class='notice'>We feel one with our surroundings.</span>")
 	owner.alpha = 200
 	START_PROCESSING(SSobj, src)
 	var/datum/role/changeling/C = owner.mind.GetRoleByType(/datum/role/changeling)
 	C.chem_recharge_slowdown += 0.25
+	var/datum/atom_hud/M = global.huds[DATA_HUD_MEDICAL]
+	M.remove_from_hud(owner)
```

Comment:
У нас в лайфе кукла нигде не добавляется обратно на худ? Какие-то функции типо update_med_hud

Reply:
Я не нашёл - но я ещё слаб в этом.

Reply:
@KIBORG04 ладно помоги я тут бессилен

Reply:
Вроде нет, костюм ниндзи вроде вызывает те-же функции.

Reply:
Не должна. Если че, он скопипастил этот код со спелла на хайд от ии, а там вроде все ок.

Reply:
Я поискал проки на худы, потом посмотрел как сделано в 2 других местах (воксостелс и нинжастелс) и решил что вариант нинжастелса подходит более. В диджитал камо не заглядывал.
В общем если будут проблемы переделаю через вариант воксостелса(99% незаметности).

Строчка
`owner.invisibility = SEE_INVISIBLE_LIVING + 1 // formal invis to prevent AI TRACKING and alt-clicking, cmon, He merged with surroundings
`
Вроде намекает на то что трекинг ИИ вроде не зависит от худов.